app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -29,12 +32,9 @@ import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.disposables.Disposable
 import io.reactivex.rxkotlin.addTo
-import kotlinx.android.synthetic.main.fragment_navigation_overlay_orig.channelsContainer
-import kotlinx.android.synthetic.main.fragment_navigation_overlay_orig.navUrlInput
-import kotlinx.android.synthetic.main.fragment_navigation_overlay_orig.settingsTileContainer
-import kotlinx.android.synthetic.main.fragment_navigation_overlay_top_nav.exitButton
-import kotlinx.android.synthetic.main.fragment_navigation_overlay_top_nav.fxaButton
-import kotlinx.android.synthetic.main.hint_bar.hintBarContainer
+import kotlinx.android.synthetic.main.fragment_navigation_overlay_orig.*
+import kotlinx.android.synthetic.main.fragment_navigation_overlay_top_nav.*
+import kotlinx.android.synthetic.main.hint_bar.*
```

Comment:
This is a bit odd but to satisfy the linter. I'm not sure how the code originally got through the linter since it was definitely failing on the wildcards.